api/routes/saved_jobs.py

```
@@ -0,0 +1,304 @@
+        # Create saved job record
+        saved_job_data = {"user_id": user_id, "job_id": job_id, "notes": request.notes}
+
+        response = supabase.table("saved_jobs").insert(saved_job_data).execute()
+
+        if response.data:
+            logger.info(f"User {user_id} saved job {job_id}")
+            return response.data[0]
+        else:
+            raise Exception("Failed to save job")
+
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)